src/protocol/basics/mod.rs

```diff
@@ -0,0 +1,4 @@
+pub mod check_zero;
+pub mod mul;
+pub mod reshare;
+pub mod reveal;
```

Comment:
The functions that each of these expose can probably be re-exported here.  That way the callers don't have to "use some::long::path;".